frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/storage-ui-source.ts

```diff
@@ -17,6 +17,18 @@ export class StorageUISource extends ObjectEnum<string> {
     DataVolumeSourceType.HTTP,
   );
   static readonly CONTAINER = new StorageUISource('Container', VolumeType.CONTAINER_DISK);
+  static readonly CLOUD_INIT = new StorageUISource('Cloud Init', VolumeType.CLOUD_INIT_NO_CLOUD);
```

Comment:
Please don't add these here. These are meant for Disk Modal to support.

You can either
-  add label property to each VolumeType and use this VolumeType label if StorageUISource is not found.
- or add support for these types here in a way disk modal and other functions which depend on StorageUISource will work correctly

Reply:
wow :+1: didn't notice that ... removed.